src/main/java/com/aetherteam/aether_genesis/blockTileEntity/HolystoneFurnaceBlockEntity.java

```diff
@@ -0,0 +1,24 @@
+package com.aetherteam.aether_genesis.blockTileEntity;
+
+import com.aetherteam.aether.blockentity.AbstractAetherFurnaceBlockEntity;
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.inventory.FurnaceMenu;
+import net.minecraft.world.item.crafting.RecipeType;
+import net.minecraft.world.level.block.state.BlockState;
+
+public class HolystoneFurnaceBlockEntity extends AbstractAetherFurnaceBlockEntity {
```

Comment:
You should be able to change this to use AbstractFurnaceBlockEntity as well, sorry I missed this I think in the feedback.

Reply:
sorry, but done